src/Hjem.tsx

```diff
@@ -75,133 +75,159 @@ const Grid = styled.div`
   margin-bottom: 15px;
   width: 80%;
   justify-content: space-around;
-`
+`;
 const ItemLocked = styled.div`
-    color: black;
-    background-color: #0000008a;
-    opacity: 0.18;
-    user-select: none;
-    font-weight: 600;
-    box-shadow: inset -1px -1px 3px black;
+  color: black;
+  background-color: #0000008a;
+  opacity: 0.18;
+  user-select: none;
+  font-weight: 600;
+  box-shadow: inset -1px -1px 3px black;
 
-    text-align: center;
-    font-size: 36px;
-    line-height: 60px;
-    border-radius: 5px;
-    margin: 5px;
-    height: 60px;
-    width: 60px;
-`
+  text-align: center;
+  font-size: 36px;
+  line-height: 60px;
+  border-radius: 5px;
+  margin: 5px;
+  height: 60px;
+  width: 60px;
+`;
 const ItemToday = styled.div`
-    color: #39506f;
-    background-color: #bfedff;
-    user-select: none;
-    font-weight: 600;
-    cursor: pointer;
+  color: #39506f;
+  background-color: #bfedff;
+  user-select: none;
+  font-weight: 600;
+  cursor: pointer;
 
-    text-align: center;
-    font-size: 36px;
-    line-height: 60px;
-    border-radius: 5px;
-    margin: 5px;
-    height: 60px;
-    width: 60px;
-`
+  text-align: center;
+  font-size: 36px;
+  line-height: 60px;
+  border-radius: 5px;
+  margin: 5px;
+  height: 60px;
+  width: 60px;
+`;
 
 const ItemUnlocked = styled.div`
-    color: white;
-    background-color: #62a2bba1;
-    cursor: pointer;
-    font-weight: 600;
+  color: white;
+  background-color: #62a2bba1;
+  cursor: pointer;
+  font-weight: 600;
 
-    text-align: center;
-    font-size: 36px;
-    line-height: 60px;
-    border-radius: 5px;
-    margin: 5px;
-    height: 60px;
-    width: 60px;
-`
+  text-align: center;
+  font-size: 36px;
+  line-height: 60px;
+  border-radius: 5px;
+  margin: 5px;
+  height: 60px;
+  width: 60px;
+`;
 
-const LogoImg =  styled.img`
-    width: 100%;
-    margin: auto;
-`
+const LogoImg = styled.img`
+  width: 100%;
+  margin: auto;
+`;
 export const Hjem = () => {
-    const d = new Date();
-    const klokkeslett = d.getHours()
-    const dato = (d.getDate() > 24) ? 0 : d.getDate();
-    const [oppgaveValgt, setOppgaveValgt] = useState<number>(0)
-    const [visSvar, setVisSvar] = useState<boolean>(false)
+  const d = new Date();
+  const klokkeslett = d.getHours();
+  const dato = d.getDate() > 24 ? 0 : d.getDate();
+  const [visSvar, setVisSvar] = useState<boolean>(false);
+
+  const { id } = useParams();
```

Comment:
Relevant endring. I stedet for å ha oppgaveValgt i state, ligger den i url-param